backend/run/copr-change-storage

```diff
@@ -0,0 +1,119 @@
+#! /usr/bin/python3
+
+"""
+Migrate existing build results for a given project and all of its CoprDirs
+from one storage (Copr backend) to another (Pulp).
+"""
+
+import os
+import sys
+import argparse
+import logging
+from copr_common.log import setup_script_logger
+from copr_backend.helpers import BackendConfigReader
+from copr_backend.storage import PulpStorage
+
+
+STORAGES = ["backend", "pulp"]
+
+log = logging.getLogger(__name__)
+setup_script_logger(log, "/var/log/copr-backend/change-storage.log")
+
+
+def get_arg_parser():
+    """
+    CLI argument parser
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--src",
+        required=True,
+        choices=STORAGES,
+        help="The source storage",
+    )
+    parser.add_argument(
+        "--dst",
+        required=True,
+        choices=STORAGES,
+        help="The destination storage",
+    )
+    parser.add_argument(
+        "--project",
+        required=True,
+        help="Full name of the project that is to be migrated",
+    )
+    parser.add_argument(
+        "--delete",
+        action="store_true",
+        default=False,
+        help="After migrating the data, remove it from the old storage",
+    )
+    return parser
+
+
+def main():
+    """
+    The main function
+    """
+    parser = get_arg_parser()
+    args = parser.parse_args()
+
+    if args.src == args.dst:
+        print("The source and destination storage is the same, nothing to do.")
+        return
+
+    if args.src == "pulp":
+        print("Migration from pulp to somewhere else is not supported")
+        sys.exit(1)
+
+    if args.delete:
+        print("Data removal is not supported yet")
```

Comment:
Since we have a pre-initialized logger, I think we should avoid using `print()` command.